doc/train/finetuning.md

```
@@ -105,13 +103,12 @@ one can select a specific branch (e.g., `CHOOSEN_BRANCH`) included in `multitask
-:::{note}
-One can check the available model branches in multi-task pre-trained model by referring to the documentation of the pre-trained model or by using the following command:
-
-```bash
-$ dp --pt show multitask_pretrained.pt model-branch
-```
-:::
+> [!NOTE]
+> One can check the available model branches in multi-task pre-trained model by referring to the documentation of the pre-trained model or by using the following command:
+>
+> ```bash
+> $ dp --pt show multitask_pretrained.pt model-branch
```

markdownlint-cli2 (0.23.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)